cfme/utils/log.py

```diff
@@ -467,6 +467,7 @@ def _configure_warnings():
     wlog.addFilter(WarningsRelpathFilter())
     wlog.addFilter(WarningsDeduplicationFilter())
     wlog.addHandler(make_file_handler('py.warnings.log'))
+    wlog.addHandler(console_handler(logging.INFO))
```

Comment:
**REQUIRED:** Does this log all INFO to the console?

Reply:
only the ones from the warning logger, which is fine

our normal logger is configured based on the other settings